test/libsolidity/syntaxTests/dataLocations/privateFunction/private_function_return_parameters_no_data_location.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    function f() private pure returns(uint[]) {}
+}
+// ----
+// TypeError: (51-57): Storage location must be "storage" or "memory" for parameter in private function, but none was given.
```

Comment:
I think we should add another case when constructing the error message such that this says "must be ... for return parameter in ...".